hrjobcontract/js/test/controllers/modal/modal-contract_test.js

```diff
@@ -0,0 +1,165 @@
+define([
```

Comment:
> //All the above steps should be done in the describe block meant for save function only. They should be done in global describe block
});
Is not implemented, as you are still not setting all the values only for `save` function. Let me know if you need help.